pkg/core/native/native_nep17.go

```diff
@@ -59,22 +59,23 @@ func newNEP17Native(name string) *nep17TokenNative {
 	n.AddMethod(md, desc)
 
 	desc = newDescriptor("balanceOf", smartcontract.IntegerType,
-		manifest.NewParameter("account", smartcontract.Hash160Type))
+		manifest.NewParameter("account", smartcontract.ByteArrayType))
```

Comment:
Very strange given there is a proposal with `Hash160` and `Uint160` type in the method itself.

Reply:
I think this needs to be discussed with core devs, obviously more specific types are expected, but for now we just need to be compatible and if that's the way C# node works, so be it for preview4.